test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -600,6 +614,14 @@ protected static void wipeAllIndices() throws IOException {
         }
     }
 
+    protected static void wipeDataStreams() throws IOException {
+        try {
+            adminClient().performRequest(new Request("DELETE", "_data_stream/*"));
+        } catch (ResponseException e) {
+            // We hit a version of ES that doesn't have data streams enabled so it's safe to ignore
```

Comment:
Maybe also check that the response code is `405`? This is what we return in the case a rest action hasn't been implemented.